django_reference_app/tests/test_model.py

```diff
@@ -0,0 +1,9 @@
+from django import test
+from django_reference_app.models import Person
+
+
+class PersonTest(test.TestCase):
+
+    def test_person_creation(self):
+        w = Person.objects.create(name='test_user', job='test_job')
```

Comment:
Single-character variable names are generally frowned upon, although acceptable in test cases when their use is short - but I'd still make it a bit more descriptive and go with `p`, if not `person`.